$NetBSD: patch-ae,v 1.3 2008/08/20 03:04:20 bjs Exp $

In pkgsrc, all Perl modules are installed into vendor_perl, not
site_perl.

--- perl/Makefile.orig	2008-08-17 16:43:06.000000000 -0400
+++ perl/Makefile
@@ -11,7 +11,7 @@ ifndef V
 endif
 
 all install instlibdir: $(makfile)
-	$(QUIET)$(MAKE) -f $(makfile) $@
+	$(QUIET)$(MAKE) -f $(makfile) INSTALLDIRS=vendor $@
 
 clean:
 	$(QUIET)test -f $(makfile) && $(MAKE) -f $(makfile) $@ || exit 0
